test/membrane_live_web/plug/auth_plug_integration_test.exs

```diff
@@ -67,36 +72,38 @@ defmodule MembraneLiveWeb.AuthPlugIntegrationTest do
     end
   end
 
-  defp test_endpoint(routing_function, conn, %{endpoint_atom: :create, body: body}) do
-    conn
-    |> post(routing_function.(conn, :create), body)
-    |> assert_status_is_401()
-  end
+  defp test_endpoint(routing_function, conn, %{endpoint_atom: atom} = endpoint_info)
+       when atom in @endpoint_atoms do
+    req_args = compute_req_args(routing_function, conn, endpoint_info)
 
-  defp test_endpoint(routing_function, conn, %{endpoint_atom: :update, body: body}) do
-    conn
-    |> put(routing_function.(conn, :update, @dummy_uuid), body)
+    atom
+    |> endpoint_atom_to_function()
+    |> apply(req_args)
     |> assert_status_is_401()
   end
 
-  defp test_endpoint(routing_function, conn, %{endpoint_atom: :delete}) do
-    conn
-    |> delete(routing_function.(conn, :delete, @dummy_uuid))
-    |> assert_status_is_401()
+  defp compute_req_args(routing_function, conn, %{endpoint_atom: atom, body: body}) do
+    [conn, compute_routing_function(routing_function, conn, atom), body]
   end
 
-  defp test_endpoint(routing_function, conn, %{endpoint_atom: :show}) do
-    conn
-    |> get(routing_function.(conn, :show, @dummy_uuid))
-    |> assert_status_is_401()
+  defp compute_req_args(routing_function, conn, %{endpoint_atom: atom}) do
+    [conn, compute_routing_function(routing_function, conn, atom)]
```

Comment:
The same call for `compute_routing_function(routing_function, conn, atom)` in both function.